viz/viz_helpers.py

```diff
@@ -45,53 +46,85 @@ def reorder_img(orig_img, reorder, by_row=True, img_size=(3, 32, 32), padding=2)
     return reordered_img
 
 
-def read_avg_kl_from_file(log_file_path, nr_latent_variables):
+def read_avg_kl_from_file(log_file_path):
     """ Read the average KL per latent dimension at the final stage of training from the log file.
-    """
-    with open(log_file_path, 'r') as f:
-        total_list = list(csv.reader(f))
-        avg_kl = [0] * nr_latent_variables
-        for i in range(1, nr_latent_variables + 1):
-            avg_kl[i - 1] = total_list[-(2 + nr_latent_variables) + i][2]
 
-    return avg_kl
+        Parameters
+        ----------
+        log_file_path : str
+            Full path and file name for the log file. For example 'experiments/custom/losses.log'. 
+    """
+    logs = pd.read_csv(log_file_path)
```

Comment:
all of this was kind of dirty, we need quick results. I think all the code is fine (although variables could be a lot more clearer, but I wrote it so I can change it) but the names of the columns `"Epoch"`, `"Value`.. should really only be defined in one place